spyder/plugins/ipythonconsole.py

```diff
@@ -877,6 +871,13 @@ def create_new_client(self, give_focus=True):
                               menu_actions=self.menu_actions)
         self.add_tab(client, name=client.get_name())
 
+        if cf is None:
+            error_msg = ("The directory {} is not writable and it is "

```

Comment:
Please add the underscore function (for translations) at the beginning of this string, like this

    error_msg = _("The directory {} ...

Else this string won't be picked up by gettext to be translated :-)